source/js/components/petition/donation-modal.jsx

```diff
@@ -67,17 +67,15 @@ class DonationModal extends React.Component {
     }
     return (
       <div className="modal-content" role="dialog">
-        <div className="modal-header text-right">
-          <button
-            className="close"
-            data-dismiss="modal"
-            aria-label="Close"
-            onClick={(e) => this.props.onClose(e)}
-            tabIndex="0"
-          >
-            <span aria-hidden="true">&times;</span>
-          </button>
-        </div>
+        <button
+          className="close"
+          data-dismiss="modal"
+          aria-label="Close"
```

Comment:
Totally unrelated, but this made me realize I had forgotten to make a few of these localizable in the Fluent branch, so thank you :)

Reply:
Thanks for catching this!